var/spack/repos/builtin/packages/py-hepdata-lib/package.py

```diff
@@ -12,6 +12,7 @@ class PyHepdataLib(PythonPackage):
     homepage = "https://github.com/HEPData/hepdata_lib"
     pypi = "hepdata_lib/hepdata_lib-0.9.0.tar.gz"
```

Comment:
Does this even build from the source code from PyPI? It looks like the tarball is missing the `requirements.txt` file.

Reply:
Yep, it does not. Fixed.